code/modules/projectiles/ammunition/ammo_casings.dm

```diff
@@ -33,7 +33,7 @@
 	desc = "A .38 bullet casing."
 	caliber = "38"
 	icon_state = "r-casing"
-	projectile_type = /obj/item/projectile/bullet/weakbullet2
+	projectile_type = /obj/item/projectile/bullet/c38
```

Comment:
this makes det revolver lethal only, are you sure this was intended?

Reply:
apologies, the change in your description had not appeared when i wrote the review

Reply:
>tweak: Detective revolver no longer can stun; it just deals damage

this was definitely intended as shown in the CL

Reply:
doing this just incentivises the detective to mod their gun right away, if it's lethal only now, why bother with the lower damage?

Reply:
It's the same thing with the captain's laser gun. He should only be using it in dire situations and when he's attacked.

Detective should *not* be placing himself in situations where he's chasing people or generally being the go-go antag hunter. He's *detective* not a security officer.

Reply:
I agree fully with that point, but only being able to retaliate lethally causes escalation. Surely the problem you describe is not a mechanical one but an administrative one?

Reply:
> I agree fully with that point, but only being able to retaliate lethally causes escalation.

The Detective's only option for fighting at range being a 25 damage lethal bullet does seem like the wrong way to solve this.
If anything this will cause them to validhunt *more* since they have a half-decent weapon, at least compared to the disablers that actual officers get.